src/styles/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  background-position: center;
+  background-image: url(/src/images/star.svg);
```

Comment:
The path to the star image should be relative to the CSS file, not the root of the project. This helps ensure that the images are loaded correctly when the project structure changes or when the project is deployed to a different environment. Use '../images/star.svg' instead of '/src/images/star.svg'.